lexicons/app/bsky/feed/getFeedGenerator.json

```diff
@@ -0,0 +1,29 @@
+{
+  "lexicon": 1,
+  "id": "app.bsky.feed.getFeedGenerator",
+  "defs": {
+    "main": {
+      "type": "query",
+      "description": "Retrieve a list of feeds created by a given actor",
+      "parameters": {
+        "type": "params",
+        "required": ["feed"],
+        "properties": {
+          "feed": {"type": "string", "format": "at-uri"}
+        }
+      },
+      "output": {
+        "encoding": "application/json",
+        "schema": {
+          "type": "object",
+          "required": ["view", "isOnline", "isValid"],
+          "properties": {
+            "view": {"type": "ref", "ref": "app.bsky.feed.defs#generatorView"},
+            "isOnline": {"type": "boolean"},
+            "isValid": {"type": "boolean"}
```

Comment:
In general we haven't been prefixing booleans with `is` (e.g. `rebase`, tooBig`, `muted`, `blockedBy`, `disabled`, `resolved`).  There is one place we do use it, though, which is on notifications `isRead`.

Reply:
Just curious, what does `isValid` check for versus `isOnline`?

Reply:
i'm down to strip out the `is` 👌

Reply:
as for the second part of this, I basically do the check against `describeFeedGenerator`, but maybe I should just be doing a sample call 🤔 

I have the code up for this now too if you want a look but basically:
- `isOnline` is "does the service respond at all"
- `isValid` is "is the did in the record correct & does this service declare that they support the given feed uri?"